glancesync/glancesync_config.py

```diff
@@ -187,6 +187,9 @@ def __init__(self, configuration_path=None, stream=None):
                 else:
                     target['only_tenant_images'] = True
 
+                target['obsolete_syncprops'] = configparser.getset(
+                        section, 'obsolete_syncprops')
```

Comment:
The boolean property was added and the documentation updated.

I've removed the file conf/glancesync.conf and add a script to generate the config file from the README.rst.